api/v1beta1/flowcollector_types.go

```diff
@@ -358,6 +358,16 @@ type FlowCollectorFLP struct {
 	// kafkaConsumerBatchSize indicates to the broker the maximum batch size, in bytes, that the consumer will accept. Ignored when not using Kafka. Default: 10MB.
 	KafkaConsumerBatchSize int `json:"kafkaConsumerBatchSize"`
 
+	//+kubebuilder:default:="30s"
+	// +optional
+	// connection update interval is the duration of time to wait between update reports of a connection
```

Comment:
is there a min/max value for setting this ? and the new field blow ?

Reply:
There are no min / max values for now. I guess we need to test deeper and even get feedback before forcing such.

@ronensc what do you think ?

Reply:
I agree. There are no min/max. Also the default values were set arbitrarily.